{{cookiecutter.project_name}}/notebooks/TriggerModelDeploy.py

```diff
@@ -45,15 +45,27 @@
 assert env != "None", "env notebook parameter must be specified"
 assert model_uri != "", "model_uri notebook parameter must be specified"
 
-{% if cookiecutter.cicd_platform == "gitHub" -%}
+{% if cookiecutter.cicd_platform in ["gitHub", "gitHubEnterprise"] -%}
+github_server = dbutils.secrets.get(
```

Comment:
We can move this logic inside the `elif` on L61 since we only use the `github_server` value in the enterprise case